README.md

```diff
@@ -165,24 +181,30 @@ python examples/<example_name>.py
 More detailed examples and tutorials are located in `notebooks` and [notebooks/README.md](notebooks/README.md) provides 
 and overview and more information. 
 
-### Contributing
 
+## Contributing to ART
+
+#### Adding new Features
 Adding new features, improving documentation, fixing bugs, or writing tutorials are all examples of helpful 
 contributions. Furthermore, if you are publishing a new attack or defense, we strongly encourage you to add it to the 
 Adversarial Robustness Toolbox so that others may evaluate it fairly in their own work.
 
 Bug fixes can be initiated through GitHub pull requests. When making code contributions to the Adversarial Robustness 
 Toolbox, we ask that you follow the `PEP 8` coding standard and that you provide unit tests for the new features.
 
+#### Validating Git Commits
 This project uses [DCO](https://developercertificate.org/). Be sure to sign off your commits using the `-s` flag or 
-adding `Signed-off-By: Name<Email>` in the commit message.
-
-#### Example
-
+adding `Signed-off-By: Name<Email>` in the commit message. Example:
 ```bash
-git commit -s -m 'Add new feature'
+git commit -s -m 'Informative commit message'
 ```
 
+#### Unit tests
+When submitting additional unit tests for ART, in order to keep the code base maintainable, please make sure each unit 
+test can run in less than 10 seconds.
```

Comment:
I'm not sure if we can limit everything to 10 seconds. How about if we say `ideally within a few seconds`?

Reply:
lol ye I wasn't sure either. I initially put it at 15 seconds but ye your suggestion is better it gets the point across without putting a hard metric